### ISSUE TITLE
What instructions should I run to evaluate the dita model on calvin (abc->d)

### ISSUE THREAD

**Issue by xjtu-hwy** [RoboDita/Dita#15]
What instructions should I run to evaluate the dita model on calvin (abc->d) (sorry I'm new to this), I've successfully run requirements_Calvs.txt and successfully downloaded the model checkpoint, I'm running torchrun --nproc_per_node=4 --nnodes=1 --node_rank=0 scripts/train_diffusion_sim.py --config-name config_diffusion_calvin batch_size=32 dataset.traj_length=11 num_pred_action=10 task_name=calvin_exp  dataset.num_given_observation=2 dataset=fix_camera use_close_loop_eval=True close_loop_eval.test_episodes_num=32  dataset.use_baseframe_action=True taskname=task_ABC_D dataname=calvin_mc close_loop_eval.eval_iters=10000  close_loop_eval.test_episodes_num=250 scheduler_type=0 wrap_grmg_data=2 +pretrained_path=dit_policy_checkpoint.pth  +use_adjust_scheduler=true lr=0.0001 epoch=15 +min_lr_scale=0.01 scheduler.warmup_epochs=1 There was a problem when num_inference_steps=10 and found that the module in Dataset_VLA could not be found

**Comment by xjtu-hwy** [comment]
Thank you for your work. I'm sorry that when I was trying to evaluate on calvin D->D, I seemed to encounter many exceptions. First of all, it prompted me
"""In 'config_diffusion_calvin': Could not find 'dataset/fix_camera"""
Then I changed """fix_camera" to "fix_traj"""

**Comment by xjtu-hwy** [comment]
Could not override 'wrap_grmg_data'.
To append to your config use +wrap_grmg_data=2
Key 'wrap_grmg_data' is not in struct
    full_key: wrap_grmg_data
    object_type=dict

**Comment by zhihou7** [comment]
Hi @xjtu-hwy many thanks for your issue. I have trained a model again for a checkpoint to reproduce the performance with 4GPUs (ABC->D). Here is the [checkpoint](https://drive.google.com/file/d/1L3X3op-oVNVVGEw8EdpiiedYp0I0gzXP/view?usp=sharing). By the way, ABCD->D achieves similar performance.

Best,

**Comment by CCCalcifer** [comment]
there's no `fix_camera.yaml` file in `config/dataset` in original code in the repository so I also meet the problem of `In 'config_diffusion_calvin': Could not find 'dataset/fix_camera'` by running the command of 
`torchrun --nproc_per_node=4 --nnodes=1 --node_rank=0 scripts/train_diffusion_sim.py --config-name config_diffusion_calvin batch_size=32 dataset.traj_length=11 num_pred_action=10 task_name=calvin_exp dataset.num_given_observation=2 dataset=fix_camera use_close_loop_eval=True close_loop_eval.test_episodes_num=32 dataset.use_baseframe_action=True taskname=task_ABC_D dataname=calvin_mc close_loop_eval.eval_iters=10000 close_loop_eval.test_episodes_num=250 scheduler_type=0 wrap_grmg_data=2 +pretrained_path=/mnt/SSD2_2T/dmt/dita_ckpt/dit_policy_checkpoint.pth +use_adjust_scheduler=true lr=0.0001 epoch=15 +min_lr_scale=0.01 scheduler.warmup_epochs=1 num_inference_steps=10`

How to deal with this problem? is this yaml necessary or I can just remove this args?

**Comment by zhihou7** [comment]
sorry for the missing files, You can just remove this. There is a default value.
